docs/docs/integrating-with-other-libraries.md

```diff
@@ -0,0 +1,266 @@
+---
+id: integrating-with-other-libraries
+title: Integrating with other libraries
+permalink: docs/integrating-with-other-libraries.html
+---
+
+An entire web application does not need to be controlled by React. React can be embedded in other applications and other applications can be embedded in React. This guide will examine some of the more common use cases, focusing on integration with jQuery and Backbone.
+
+## Integrating with DOM Manipulation Plugins
+
+React is unaware of changes made to the DOM outside of React. It determines updates based on its own internal representation, and if those updates are invalidated, React has no way to recover.
+
+This does not mean it is impossible or even necessarily difficult to combine React with other ways of affecting the DOM, you just have to be mindful of what each are doing.
+
+The easiest way to avoid conflicts is to prevent the React component from updating. This can be done explicitly by returning false from [`shouldComponentUpdate()`](/react/docs/react-component.html#shouldcomponentupdate), or by rendering elements that have no reason to change.
+
+```js
+class SomePlugin extends React.Component {
+  componentDidMount() {
+    this.$el = $(el);
+    this.$el.somePlugin();
+  }
+
+  componentWillUnmount() {
+    this.$el.somePlugin('destroy');
+  }
+
+  render() {
+    return <div ref={el => this.el = el} />
+  }
+}
+```
+A [ref](/react/docs/refs-and-the-dom.html) is used to pass the underlying DOM element to the plugin. The `<div>` element has no properties or children, so React has no reason to update it.
```

Comment:
Worth highlighting this a bit somehow. The key is that rendering a consistent set of elements each time means React won't be trying to make changes, so a lib that _does_ modify the DOM can get away with it and not clash with React applying updates.